python/ray/_private/job_manager/job_manager.py

```diff
@@ -176,7 +214,8 @@ class JobManager:
     as lost once the ray cluster running job manager instance is down.
     """
     JOB_ACTOR_NAME = "_ray_internal_job_actor_{job_id}"
-    START_ACTOR_TIMEOUT_S = 10
+    # Time given to setup runtime_env for job supervisor actor.
+    START_ACTOR_TIMEOUT_S = 60
```

Comment:
This is probably still not a safe timeout (some conda envs / container pulls could take a really long time). Do we need a timeout at all?

Reply:
See below, I think we should just remove the timeout for now

Reply:
Oh, that reminds me -- we don't properly handle stopping jobs that are `PENDING` forever right now... not too worried about this case at the moment but something to keep in mind.